### PR TITLE
Added Tiberian Sun railroad track rendering

### PR DESCRIPTION
**Pull request by Mailaender**
For now this is just doing the railroad track rendering. Already comes with a test case (see shellmap) and I updated https://github.com/Mailaender/OpenRA/compare/ts-map-import as well to show off that this is legacy compatible:

image

### CODE REVIEW

**Comment by pchote** [comment]
This doesn't implement the custom terrain type that will be needed for train pathing.  I don't think that should block us from importing these as cosmetic map props, so :+0.5: (haven't tested this ingame, but I'm okay with the concept).

Having multiple custom terrain layers for different prop types kind of sucks, so I suggest we leave that until after #8916, which I expect (for the implementation I have in my head at least) will make it possible to use the same layer for both bridges and railroads.

**Comment by pchote** [comment]
The z-offsets need adjusting so that these render below actors.

screen shot 2015-09-01 at 05 38 56

**Comment by Mailaender** [comment]
Rebased and fixed the ZOffset.

File: mods/ts/sequences/misc.yaml [hunk 1]
```diff
@@ -489,3 +489,83 @@ palet03:
 palet04:
 	idle:
 
+tracks01:
+	idle:
+		UseTilesetExtension: true
+		ZOffset: -1c511
```
Comment (pchote) [inline mods/ts/sequences/misc.yaml]: Why 1.5 cells? 0.5 should be sufficient.

Comment (Mailaender) [inline mods/ts/sequences/misc.yaml]: 0.5 isn't:

image

1.5 is:

image

**Comment by dan9550** [comment]
Hooray, i won't have track bits mission on my shellmap after this.

**Comment by penev92** [comment]
:+1:

File: mods/ts/sequences/misc.yaml (hunk 1 above)
Comment (pchote) [inline mods/ts/sequences/misc.yaml]: What about 1.0?

Comment (Mailaender) [inline mods/ts/sequences/misc.yaml]: Seems to be enough. Updated.

**Comment by abcdefg30** [comment]
Looks good to me, too. :+1: / :white_check_mark:

**Pull request merged**